openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMBinaryClassificationModel.java

```diff
@@ -109,7 +109,7 @@ public void close() throws Exception {
      * @return the feature contributions.
      * @since 1.2.2
      */
-    public double[] getFeatureContributions(final Instance instance) {
+     double[] getFeatureContributions(final Instance instance) {
```

Comment:
isn't this a breaking change?

Reply:
Also had the same insecurity.
according to @shenggwang it isn't because this is a lib and an api package.

Reply:
but we don't know where this is being used or by who, right ?